apps/omg_status/lib/omg_status/metric/event.ex

```diff
@@ -48,6 +48,11 @@ defmodule OMG.Status.Metric.Event do
   """
   def name(:block_subbmission), do: "block_subbmission_gas"
```

Comment:
Not directly related but it's in view...

```suggestion
  def name(:block_submission), do: "block_submission_gas"
```